shell/monitor/monitor_adla.sh

```diff
@@ -16,205 +16,62 @@ _monitor_ADLA(){
 
     output=$(az dla job show --output json --account "${adla_account}" --job-identity "${job_id}")
 
-    jq -c --arg exp_id "${experiment_id}" '[
-      { index: { "_id": .jobId } },
-      {
-          id: .jobId,
-          experimentId: $exp_id,
-          name,
-          degreeOfParallelism,
-          priority,
-          submitTime,
-          startedTime: .startTime,
-          finishedTime: .endTime,
-          finalStatus: .result,
-          state,
-          runtimeVersion: .properties.runtimeVersion,
-          totalCompilationTime: .properties.totalCompilationTime,
-          totalPauseTime: .properties.totalPauseTime,
-          totalQueuedTime: .properties.totalQueuedTime,
-          totalRunningTime: .properties.totalRunningTime
-      }
-    ] | .[]' <<< "$output"
+    if [[ ! "$output" =~ (^(null|\{\})$|^$) ]]; then
+
+      jq -c --arg exp_id "${experiment_id}" '[
+        { index: { "_id": .jobId } },
+        {
+            id: .jobId,
+            experimentId: $exp_id,
+            name,
+            degreeOfParallelism,
+            priority,
+            submitTime,
+            startedTime: .startTime,
+            finishedTime: .endTime,
+            finalStatus: .result,
+            state,
+            runtimeVersion: .properties.runtimeVersion,
+            totalCompilationTime: .properties.totalCompilationTime,
+            totalPauseTime: .properties.totalPauseTime,
+            totalQueuedTime: .properties.totalQueuedTime,
+            totalRunningTime: .properties.totalRunningTime
+        }
+      ] | .[]' <<< "$output"
+    fi
 
   done
 
 }
 
-# test
-#_monitor_ADLA 12345
-
+<<<<<<< HEAD
```

Comment:
merge conflict marker here

Reply:
Sorry, this got commited by mistake. It should have not.